### PR TITLE
Add format task

### PR DESCRIPTION
**Pull request by anxolin**
Adds a task for formatting the code easier

Test:
```bash
yarn format
```

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr24--dexreact.review.gnosisdev.com

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
     "lint:fix": "eslint --fix . --ext .ts,.tsx,.js",
     "stats": "NODE_ENV=production webpack --mode production --env.stats --profile --json > stats.json && webpack-bundle-analyzer ./stats.json ./dist",
     "test": "echo Testing... && echo OK",
-    "verify": "npm run lint && npm run test"
+    "verify": "npm run lint && npm run test",
+    "format": "prettier --write \"src/**/*.ts\" \"src/**/*.js\""
```
Comment (fleupold) [inline package.json]: Should this run as part of the hooks?

Comment (anxolin) [reply package.json]: The lint already checks that the code is OK, we autoformat in the code. 
This format won't be used unless copy pasting code/files for example

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr24--dexreact.review.gnosisdev.com

**Pull request merged**